src/main/java/org/folio/service/inventory/InventoryInstanceManager.java

```diff
@@ -381,28 +406,48 @@ public Future<SharingInstance> createShadowInstanceIfNeeded(String instanceId, S
       });
   }
 
-  private Future<String> shareInstanceAmongTenantsIfNeeded(String instanceId, ConsortiumConfiguration consortiumConfiguration,
-                                                           List<Location> locations, RequestContext requestContext) {
-    return findTenantsWithUnsharedInstance(instanceId, locations, requestContext)
-      .map(tenantIds -> tenantIds.stream().map(targetTenantId -> sharingInstanceService.createShadowInstance(instanceId,
-        targetTenantId, consortiumConfiguration, requestContext)).toList())
-      .compose(HelperUtils::collectResultsOnSuccess)
-      .map(sharingInstances -> instanceId);
+  public static String getPublisher(JsonObject instance) {
+    JsonArray publication = instance.getJsonArray(INSTANCE_PUBLICATION);
```

Comment:
small suggestion to use var. type is clear, since we are getting getJsonArray()

Reply:
done